spec/helpers/ruby_version_spec.rb

```diff
@@ -117,32 +131,42 @@
     end
   end
 
-
   it "detects non mri engines" do
-    Hatchet::App.new("ruby_193_jruby_173").in_directory_fork do |dir|
+    Hatchet::App.new("default_ruby").in_directory_fork do |_|
+      dir = Pathname(Dir.pwd)
       Bundler.with_unbundled_env do
-        ruby_version   = LanguagePack::RubyVersion.new(@bundler.install.ruby_version, is_new: true)
-        version_number = "1.9.3"
-        engine_version = "1.7.3"
-        engine         = :jruby
-        version        = "ruby-#{version_number}-#{engine}-#{engine_version}"
-        to_gemfile     = "ruby '#{version_number}', :engine => '#{engine}', :engine_version => '#{engine_version}'"
-        expect(ruby_version.version_without_patchlevel).to eq(version)
+        dir.join("Gemfile").write(<<~EOF)
+          source "https://rubygems.org"
+
+          ruby '2.6.8', engine: 'jruby', engine_version: '9.3.6.0'
+        EOF
+        dir.join("Gemfile.lock").write(<<~EOF)
+          GEM
+            remote: https://rubygems.org/
+            specs:
+          PLATFORMS
+            java
+
+          DEPENDENCIES
+
+          RUBY VERSION
+            ruby 2.6.8p001 (jruby 9.3.6.0)
+
+          BUNDLED WITH
+            2.3.25
+        EOF
+
+        ruby_version   = LanguagePack::RubyVersion.new(
+          @bundler.install.ruby_version,
+          is_new: true
+        )
+        version_number = "2.6.8"
+        engine_version = "9.3.6.0"
+        engine = :jruby
+        expect(ruby_version.version_without_patchlevel).to eq("ruby-#{version_number}-#{engine}-#{engine_version}")
         expect(ruby_version.engine_version).to eq(engine_version)
-        expect(ruby_version.to_gemfile).to eq(to_gemfile)
```

Comment:
Same as above, update to not rely on a github template. Update to use a more recent Ruby version